src/backend/api/controllers/reviewControllers.js

```diff
@@ -0,0 +1,147 @@
+const knex = require("../../database");
+
+const getAllReviews = async (req, res) => {
+    try {
+        const reviews = await knex
+        .from("review")
+        .select(
+            "id", 
+            "title",
+            "description",
+            "meal_id",
+            "stars",
+            "created_date"
+        );
+
+        reviews.length === 0
+        ? res.status(404).json({"error": "Reviews not found"})
+        : res.status(400).json(reviews);
+    } catch (error) {
+        res.status(500).json(error);
+    }
+};
+
+const getMealReviews = async (req, res) => {
+    try {
+        const id = req.params.meal_id;
+
+        const rows = await knex
+        .from("review")
+        .select(
+            "review.title" , 
+            "review.description", 
+            "stars", 
+            "meal.title AS meal", 
+            "location",
+            "review.created_date"
+            )
+        .leftJoin("meal", {"meal.id": "meal_id"})
```

Comment:
Well, In my perception I `meal_id` should be `review.meal_id`. I'm not so sure, and yes, according to the syntax it should `review.meal_id`.